src/coreclr/src/System.Private.CoreLib/src/System/String.CoreCLR.cs

```diff
@@ -26,6 +26,7 @@ public partial class String
         [System.Runtime.CompilerServices.IndexerName("Chars")]
```

Comment:
Nit: You can remove it here as well while you are on it

Reply:
Fixed in 7th commit.